modules/release-notes/pages/index.adoc

```diff
@@ -102,6 +111,8 @@ vertex function can also be used with vertex input parameters in the query-body
 
 === GraphStudio
 
+* [3.9.3] Added openCypher syntax support to xref:gui:graphstudio:write-open-cypher-queries-in-tigergraph.adoc[].
```

Comment:
You don't have any text in the brackets at the end [], so it will default to the page title.  That would read a little oddly.  Also, I think you are focusing on the wrong thing. "added openCypher syntax support" is a GSQL improvement; GraphStudio is just a wrapper on GSQL.  What did GraphStudio do?  Added an openCypher option to the Add New Query dialog, to simplify writing and running openCypher in GraphStudio.

Reply:
changed this with latest commit.